code/crates/core-consensus/src/state.rs

```diff
@@ -51,7 +51,7 @@ where
             ctx,
             driver,
             params,
-            input_queue: Default::default(),
+            input_queue: BoundedQueue::new(100), // FIXME
```

Comment:
Do you plan to make it a config parameter?

Reply:
Ah right, forgot about that. Thanks!

Reply:
https://github.com/informalsystems/malachite/pull/1091/commits/51dc7912dbe9af536839579abf5d50c684ad1627